template.yaml

```diff
@@ -0,0 +1,26 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Resources:
+  Tachyon:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: ./
+      Handler: dist/lambda-handler.handler
+      Environment:
+        Variables:
+          S3_BUCKET: hmn-uploads
```

Comment:
Can this use a dummy name if it's just a template? Don't want to risk it accidentally being used

Reply:
This is meant to be used to be able to do the integration test. There's no permissions to read / write the bucket it's just public

Reply:
I see. Can we use the test bucket in that case? (Per compliance, not using prod data for testing/development.)

Reply:
this is no different than having a test which does `curl http://humanmade.com/robots.txt` to check it's working, so I don't think that really falls under using prod data for testing / development. That's more to do with copying prod data to development, so it leaks data potentially to people who only have access to "test data".